Assets/MixedRealityToolkit/Utilities/CameraCache.cs

```diff
@@ -30,7 +30,12 @@ public static Camera Main
                     switch (cameras.Length)
                     {
                         case 0:
-                            return null;
+                            Debug.LogWarning("No main camera found. The mixed reality toolkit requires at least one camera in the scene. One will be generated now.");
+                            GameObject mainCameraGo = new GameObject("Main Camera");
+                            Camera newMainCamera = mainCameraGo.AddComponent<Camera>();
+                            mainCamera = Refresh(newMainCamera);
```

Comment:
should we go ahead and add the `MainCamera` tag here as well?

Reply:
Tagging the camera causes a _SENDMESSAGE CANNOT BE CALLED DURING AWAKE_ warning. I've removed it until I can figure out what Unity is doing under the hood with tag assignment. Pretty weird.

Reply:
Fixed by passing tag during creation.